conda/recipes/cuvs/meta.yaml

```
@@ -69,6 +69,7 @@ requirements:
     - libcuvs {{ version }}
     - python x.x
     - rmm ={{ minor_version }}
+    - numpy >=1.23,<3.0a0
```

Somewhat tangential, but see `cuda-python` in the wheel as well. So think we can add it here too

Not seeing any other differences

```suggestion
    - cuda-python
    - numpy >=1.23,<3.0a0
```